src/files/test_file_reading.py

```diff
@@ -59,7 +60,7 @@ def test_files_open():
     assert file.closed
 
     # Open file using with.
-    with open('src/files/multi_line_file.txt', 'r') as file:
+    with open(path + 'multi_line_file.txt', 'r') as file:
```

Comment:
Could you also try opening this fine in binary mode `rb`?

Reply:
done